spec/requests/route_spec.rb

```diff
@@ -7,63 +7,44 @@
 
   let(:app) { Sinatra::Application }
   let!(:user) { authenticated }
+  let!(:journey1) { create(:journey, annual_miles: 10000) }
+  let!(:journey2) { create(:journey, annual_miles: 20000) }
+  let!(:commitment) { create(:commitment, user: user, journey: journey1) }
+  let!(:route) { create(:route, user: user, commitment: commitment, mileage: 12000) }
 
-  describe 'POST /user/routes' do
```

Comment:
Any particular reason why you removed these tests? Your code changed are in the `PATCH` route, these wouldn't need to be changed.